languagetool-language-modules/en/src/main/java/org/languagetool/language/CanadianEnglish.java

```diff
@@ -67,4 +83,11 @@ public List<Rule> getRelevantLanguageModelCapableRules(ResourceBundle messages,
     return rules;
   }
 
+  public static @NotNull CanadianEnglish getInstance() {
+    Language language = Objects.requireNonNull(Languages.getLanguageForShortCode(LANGUAGE_SHORT_CODE));
+    if (language instanceof CanadianEnglish canadianEnglish) {
```

Comment:
This will not work with the close source part :( See my comments in German: https://github.com/languagetool-org/languagetool/blob/482657017ccde7123292c801ef9f332a4f6c9e80/languagetool-language-modules/de/src/main/java/org/languagetool/language/GermanyGerman.java#L90

This applies for those languages and their variants:

de
en
es
pt
fr
nl

Reply:
fixed